.azure-pipelines/templates/osx/build-and-unit-test.yml

```diff
@@ -30,3 +30,14 @@ steps:
       testRunTitle: Mac $(configuration) Unit Tests
       publishRunAttachments: true
     condition: succeededOrFailed()
+
+  - script: mkdir $(Build.ArtifactStagingDirectory)/screencap && screencapture $(Build.ArtifactStagingDirectory)/screencap/img.png
+    displayName: Take screencapture
+    condition: always()
+
+  - task: PublishPipelineArtifact@1
+    displayName: Publish screencapture
+    inputs:
+      targetPath: $(Build.ArtifactStagingDirectory)/screencap
+      artifactName: Screencap_macOS_$(configuration)
+    condition: always()
```

Comment:
I'm guessing this is just for testing, to see if we have a popup blocking our builds?

Reply:
Correct. I'm going to be iterating on the PR to try and discover what's going on with the hanging Scalar.app tests. You might want to unsubscribe :)